File: Components/WinForms/CS/ChartsMainDemo/Modules/AreaViews/3DSplineArea.cs
namespace DevExpress.XtraCharts.Demos.AreaViews {
    public partial class SplineArea3dDemo : Area3dDemoBase {
        ChartControl chart;

        public override ChartControl ChartControl { get { return this.chart; } }

        public SplineArea3dDemo() {
            InitializeComponent();
        }
        protected override void InitControls() {
            base.InitControls();
            ChartControl.DataSource = DevAV.GetOutsideVendorCosts();
        }
    }
}

File: Components/WinForms/VB/ChartsMainDemo/Modules/AreaViews/3DSplineArea.vb
Imports Microsoft.VisualBasic
Imports System
Namespace DevExpress.XtraCharts.Demos.AreaViews
	Partial Public Class SplineArea3dDemo
		Inherits Area3dDemoBase
		Private chart As ChartControl

		Public Overrides ReadOnly Property ChartControl() As ChartControl
			Get
				Return Me.chart
			End Get
		End Property

		Public Sub New()
			InitializeComponent()
		End Sub
		Protected Overrides Sub InitControls()
			MyBase.InitControls()
			ChartControl.DataSource = DevAV.GetOutsideVendorCosts()
		End Sub
	End Class
End Namespace

